UPGRADING.md

```diff
@@ -41,6 +41,19 @@ Prior to this version the response would be `one is missing`.
 
 See [#1510](https://github.com/ruby-grape/grape/pull/1510) for more information.
 
+#### Change default status code of DELETE to 204
+
+**Breaking change**: Sets the default response status code for a delete request to 204.
+Doing it makes the response more explicit, cause a delete request returns in most cases an empty body – the resource was deleted/voided. By setting it to 204, other as 200, makes the handling on consumer side easier.
```

Comment:
How about _A status of 204 makes the response more distinguishable and therefore easier to handle on the client side, particularly because a DELETE request typically returns an empty body as the resource was deleted or voided._